typescript/ecs/fargate-service-with-auto-scaling/index.ts

```diff
@@ -0,0 +1,50 @@
+import appautoscaling = require('@aws-cdk/aws-applicationautoscaling');
+import ecs = require('@aws-cdk/aws-ecs');
+import ec2 = require('@aws-cdk/aws-ec2');
+import iam = require('@aws-cdk/aws-iam');
+import cdk = require('@aws-cdk/cdk');
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-fargate-application-autoscaling');
```

Comment:
Can we please rewrite it more idiomatically by defining a class that extends `cdk.Stack` and always use `this` as the `scope` argument.

Reply:
done.